### PR TITLE
Bump Packages, Update Tests

### LINKED ISSUE #113: Python 3.11?
It looks like naplam-panos has an issue running within Python3.11. Is there going to be a update to get it supported for 3.11?

### PR DESCRIPTION
**Pull request by housepbass**
Closes #113

### CODE REVIEW

**Comment by housepbass** [comment]
@itdependsnetworks Can you drop Python 3.6-7 from CI?

**Comment by itdependsnetworks** [comment]
i think you just need to update the ci.yml, here as an example (but check for other places) https://github.com/napalm-automation-community/napalm-panos/blob/develop/.github/workflows/ci.yml#L78

File: pyproject.toml [hunk 1]
```diff
@@ -27,23 +26,22 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = ">=3.8,<3.12"
 cryptography = "3.3.2"
-napalm = "^3.0"
-lxml = "4.6.5"
+napalm = "4.1.0"
```
Comment (itdependsnetworks) [inline pyproject.toml:31]: should we just do ^4?

File: pyproject.toml [hunk 2]
```diff
@@ -27,23 +26,22 @@ include = [
...
+lxml = "5.2.1"
```
Comment (itdependsnetworks) [inline pyproject.toml:32]: I get that this was static before, but should be a bit more loose.

File: pyproject.toml [hunk 3]
```diff
@@ -27,23 +26,22 @@ include = [
...
 pan-python = "*"
-netmiko = "^3.3.2"
+netmiko = "4.3.0"
```
Comment (itdependsnetworks) [inline pyproject.toml:34]: Should keep this loose as well

File: pyproject.toml (hunk 1 above)
Comment (housepbass) [reply pyproject.toml]: Updated

File: pyproject.toml (hunk 2 above)
Comment (housepbass) [reply pyproject.toml]: Updated

**Comment by itdependsnetworks** [comment]
Can you update changelog, should include #105, #108, and this one.

**Pull request merged**